### PR TITLE
Use cloudfront secure headers

### PR DESCRIPTION
**Pull request by ConnorKirk**
*Issue #, if available:*
#40 

*Description of changes:*

This PR updates the sample to use CloudFront Response Headers, rather than a Lambda at Edge function to add Security Headers to responses.

- [x] Add CloudFront ResponseHeaderPolicy to template
- [x] Remove Lambda At Edge from template
- [x] Remove other references to lambda at edge from sample (readme, CI etc)
- [x] Update README and architecture diagram

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,7 +43,10 @@ package-static:
 	make build-static
 	cd source/witch && zip -r ../../witch.zip nodejs
 
-package-function:
-	make clean
-	make package-static
-	cd source/secured-headers/ && zip -r ../../s-headers.zip index.js
+
```
Comment (HieronymusLex) [inline Makefile:46]: mmm delicious whitespace

File: README.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ The following diagram shows an overview of how the solution works:
 1. The viewer requests the website at www.example.com.
 2. If the requested object is cached, CloudFront returns the object from its cache to the viewer.
 3. If the object is not in CloudFront’s cache, CloudFront requests the object from the origin (an S3 bucket).
-4. S3 returns the object to CloudFront, which triggers the Lambda@Edge origin response event.
-5. The object, including the security headers added by the Lambda@Edge function, is added to CloudFront’s cache.
+4. S3 returns the object to CloudFront
+5. The object is added to CloudFront’s cache.
```
Comment (HieronymusLex) [inline README.md:23]: nit: these instructions switch between active/passive voice

**Comment by HieronymusLex** [comment]
the diagram should be using the most up to date architecture icons

File: Makefile (hunk 1 above)
Comment (ConnorKirk) [reply Makefile:46]: Guilty! I will tidy it up. 😂

**Comment by ConnorKirk** [comment]
I've updated the diagram to use the latest icon set

**Review by svozza — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -14,30 +14,30 @@ For more information about each of these components, see the **Solution details*
...
 
 1. The viewer requests the website at www.example.com.
 2. If the requested object is cached, CloudFront returns the object from its cache to the viewer.
 3. If the object is not in CloudFront’s cache, CloudFront requests the object from the origin (an S3 bucket).
-4. S3 returns the object to CloudFront, which triggers the Lambda@Edge origin response event.
-5. The object, including the security headers added by the Lambda@Edge function, is added to CloudFront’s cache.
-6. (Not shown) The objects is returned to the viewer. Subsequent responses for the object are served from the CloudFront cache.
+4. S3 returns the object to CloudFront
+5. CloudFront caches the object.
+6. The objects is returned to the viewer. Subsequent responses for the object are served from the CloudFront cache.
```
Comment (HieronymusLex) [inline README.md:24]:
```suggestion
6. The object is returned to the viewer. Subsequent responses for the object are served from the CloudFront cache.

```

**Pull request merged**